client/grpc/client.go

```diff
@@ -0,0 +1,50 @@
+package grpc
+
+import (
+	"errors"
+
+	"github.com/kava-labs/kava/client/grpc/query"
+	"github.com/kava-labs/kava/client/grpc/util"
+)
+
+// KavaGrpcClient enables the usage of kava grpc query clients and query utils
+type KavaGrpcClient struct {
+	config KavaGrpcClientConfig
+
+	// Query clients for cosmos and kava modules
+	Query *query.QueryClient
+
+	// Utils for common queries (ie fetch an unpacked BaseAccount)
+	*util.Util
+}
+
+// KavaGrpcClientConfig is a configuration struct for a KavaGrpcClient
+type KavaGrpcClientConfig struct {
+	// note: add future config options here
+}
+
+// NewClient creates a new KavaGrpcClient via a grpc url
+func NewClient(grpcUrl string) (*KavaGrpcClient, error) {
```

Comment:
recommend adding url as a field to the config struct above and taking that as the only input to this function